packages/mobile/src/components/core/Button.tsx

```diff
@@ -267,21 +271,32 @@ export const Button = (props: ButtonProps) => {
   const { primaryDark1, neutralLight10, neutralLight7, accentRedDark1 } =
     useThemeColors()
 
-  const pressColor = {
-    primary: primaryDark1,
-    secondary: primaryDark1,
-    common: primaryDark1,
-    commonAlt: neutralLight10,
-    destructive: accentRedDark1
-  }
+  const pressColor = useMemo(() => {
+    // If a custom color is specified
+    // darken the color by 10% for the press state
+    if (customColor) {
+      const c = new Color(customColor)
+      return c.darken(0.1).hex()
```

Comment:
I've used polish is in the past and have really liked it, curious what you think

Reply:
polished looks really nice but we already have a dependency on `color` here https://github.com/AudiusProject/audius-client/blob/main/packages/mobile/src/utils/colorizeLottie.ts#L11 so opted to use that

Maybe we could update `colorizeLottie` to use polish?

Reply:
Good to know we already use color, yeah possibly! It would be useful for the glassmorphic now playing, maybe we can update then